<launch>
  <node name="serial_232" pkg="serial_control" type="serial_232" output="screen">
    <param name="port" value="/dev/ttyUSB1" />
    <param name="baudrate" value="115200" />
  </node>
</launch>
